e2e/MoviesView.test.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-undef */
+const puppeteer = require('puppeteer');
+const regeneratorRuntime = require('@babel/runtime/regenerator');
+
+let browser;
+let page;
+
+beforeAll(async () => {
+  browser = await puppeteer.launch({
+    headless: false,
+    slowMo: 0,
+  });
+  page = await browser.newPage();
+});
+
+describe('MoviesView page', () => {
+  const moviesURL = 'http://localhost:8888/movies';
+  const movieCardSelector = '[data-testid="movieCard"]';
+  const nextPageSelector = '[data-testid="nextPage"]';
+  const movieInfoSelector = '[data-testid="movieInfo"]';
+  const activePageSelector = '[data-testid="active"]';
+  const moviesErrors = [];
+
+  it('loads correctly', async () => {
+    await page.goto(moviesURL);
+    await page.waitForSelector(movieCardSelector);
+    const movieCard = await page.$(movieCardSelector);
+    expect(movieCard).toBeTruthy();
```

Comment:
I think check if `movieCard` > 0 will be better than check only one card. WDYT?

Reply:
Maybe it's the same?
Now I check if any movieCard exist, and then I will check movieCard length is grater than 0.